### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by Street-Philosopher**
first few files of the Italian translation. It has all of the files of src copied in a separate folder (`translations/it/`), but only some have been translated already

### CODE REVIEW

**Review by avivace — CHANGES_REQUESTED** [review]
@ISSOtm Are we sure we need the images copied over when submitting translations?

**Comment by avivace** [comment]
Hey @Street-Philosopher , we finally moved to Crowdin and I started porting your translation there. Would it be possible for you to create and account there and give me your name? It's way easier to contribute to translations there, gradually approving those and being able to update them with time. 

https://crowdin.com/project/gb-asm-tutorial

File: translations/it/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Struttura del tutorial
```
Comment (avivace) [inline translations/it/roadmap.md:1]: note for translators: this file has been ported to crowdin.

File: translations/it/part1/toolchain.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Gli strumenti
```
Comment (avivace) [inline translations/it/part1/toolchain.md:1]: note for translators: this file has been ported to crowdin.

File: translations/it/help-feedback.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Aiuto e suggerimenti
```
Comment (avivace) [inline translations/it/help-feedback.md:1]: note for translators: this file has been ported to crowdin.

File: translations/it/part1/assembly.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+La CPU ha un numero di istruzioni limitato, quindi non tutte le combinazioni di parametri sono possibili, né per `ld` né per le altre mnemoniche. Ne parleremo meglio quando inizieremo il codice vero e proprio.
+
+:::
+
+::: tip:🤔
+
+RGBDS ha una pagina di [riferimento per le istruzioni](https://rgbds.gbdev.io/docs/gbz80.7) che vale la pena salvare, e che può essere consultata localmente col comando `man 7 gbz80` se RGBDS è installato sul tuo sistema (eccetto windows...).
+Le spiegazioni sono molto brevi: non è inteso come un tutorial quanto più come un promemoria.
+
+:::
```
Comment (avivace) [inline translations/it/part1/assembly.md:58]: note for maintainers: ported to crowdin till here

**Comment by Street-Philosopher** [comment]
Hi @avivace very sorry if it took me this long to respond.

I just signed up. My username is still `Street-Philosopher`. Thank you!

**Comment by avivace** [comment]
> Hi @avivace very sorry if it took me this long to respond.
> 
> I just signed up. My username is still `Street-Philosopher`. Thank you!

Amazing, thanks a lot! I just invited you to the project. I hope that's clear enough for you to start contributing there. I ported most of your translations you made in this PR. Feel free to come back for any doubt (here on in another issue).

I will close this PR.

**Pull request closed without merging**